vhd/aib/AKSeImageTemplateWindows.json

```diff
@@ -105,6 +105,12 @@
                     "version": "17763.7009.250306"
                 },
                 "customize": [
+                    {
+                        "type": "File",
+                        "name": "download signedscripts",
+                        "sourceUri": "https://akseartifacts.blob.core.windows.net/windows/provisioning/signedscripts-v0.0.18.zip",

```

Comment:
Does this work with anonymous pull disabled?

Reply:
Yes